pkg/scheduler/api/node_info.go

```diff
@@ -163,15 +163,18 @@ func (ni *NodeInfo) allocateIdleResource(ti *TaskInfo) error {
 		ni.Idle.Sub(ti.Resreq)
 		return nil
 	}
-	ni.State = NodeState{
```

Comment:
Removing this because I do not want to modify node info state on error. I think, if it is necessary, this state could be set explicitly after receiving an error, depending on the context in which error occurred.